Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -93,9 +93,20 @@ void VideoConfig::Load(const std::string& ini_file)
 	enhancements->Get("PostProcessingShader", &sPostProcessingShader, "");
 	enhancements->Get("StereoMode", &iStereoMode, 0);
 	enhancements->Get("StereoDepth", &iStereoDepth, 20);
-	enhancements->Get("StereoConvergence", &iStereoConvergence, 20);
+	enhancements->Get("StereoConvergencePercentage", &iStereoConvergencePercentage, 100);
 	enhancements->Get("StereoSwapEyes", &bStereoSwapEyes, false);
 
+	// Some users still have absolute convergence values in their config,
+	// convert those to percentage values.
+	int old_convergence;
+	enhancements->Get("StereoConvergence", &old_convergence, -1);
+	if (old_convergence != -1)
+	{
+		iStereoConvergencePercentage = (old_convergence * 100) / 20;
+		enhancements->Set("StereoConvergencePercentage", iStereoConvergencePercentage);
+		enhancements->Delete("StereoConvergence");
```

Comment:
No, we don't really have anything like that. Either do it here or in some place where the INI often gets saved (the settings dialog?)

By the way, I'm not sure if completely deleting the old setting is the best way to go. It creates problems for users who move between new and old builds. For the reference, the GCMPathes to ISOPaths migration didn't do it.